packages/app/components/Chatbot/Chatbot.tsx

```diff
@@ -44,28 +43,14 @@ export const ChatbotComponent: React.FC = () => {
   const profile = useProfile()
   const [isLoading, setIsLoading] = useState(false)
   const [interactionId, setInteractionId] = useState<number | null>(null)
-  const [preDeterminedQuestions, setPreDeterminedQuestions] = useState<
-    PreDeterminedQuestion[]
-  >([
-    {
-      question: 'Can each table have have multiple foreigns keys?',
-      answer: 'Yes, each table can have multiple foreign keys.',
-    },
-    {
-      question: 'What is cardinality, explain related information too',
-      answer:
-        'Cardinality refers to the maximum number of relationship instances that an entity can have in a relationship. It can be one-to-one (1:1), one-to-many (1:* or 1:N), or many-to-many (*:* or N:M) depending on the relationship type.',
-    },
-    {
-      question: 'When and where is the final?',
-      answer: 'December 16, 2023 8:30AM	in COM 201',
-    },
-  ])
+  const [preDeterminedQuestions, setPreDeterminedQuestions] =
```

Comment:
lint comment

Reply:
set is not used here, but preDeterminedQuestions is used. In the next change, it will be used again so we can leave it for now?